### PR TITLE
Lp 481 Allow markdown links in item descriptions

### PR DESCRIPTION
**Pull request by sarah-cul**
Allow markdown links in item descriptions I added a new presenter class for fields, this presenter class is used in various .erb files that render the items description. Tested under Home, Pages, About, Browse, and Search

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -49,5 +49,11 @@ class CatalogController < ApplicationController
     config.add_results_collection_tool(:view_type_group)
 
     config.view.masonry.title_only_by_default = true
+
+    config.add_index_field(
+      'spotlight_upload_description_tesim',
```
Comment (chrisrlc) [inline app/controllers/catalog_controller.rb:54]: Minor, but maybe we could swap this string for `Spotlight::Engine.config.upload_description_field` to allow that field name to change if we ever decide?

Comment (sarah-cul) [reply app/controllers/catalog_controller.rb]: great idea!

File: app/helpers/spotlight_helper.rb [hunk 2]
```diff
@@ -11,4 +11,13 @@ def display_exhibit_published_date(exhibit)
 
     exhibit.published_at? ? l(exhibit.published_at, format: :long) : 'Unknown'
   end
+
+  def render_markdown_links(options = {})
+    values = options[:value]
+    return ''.html_safe unless values
+    safe_html = values.map do |value|
+      ERB::Util.html_escape(value).gsub(/\[([^\[]+)\]\(([^\)]+)\)/, '<a href="\2">\1</a>')
```
Comment (chrisrlc) [inline app/helpers/spotlight_helper.rb:19]: Looks good! I could see us wanting to expand available markdown in this field in the future by rendering it the way we seem to for the widgets, i.e. using something like `GitHub::Markup.render('.md', value).html_safe` from Spotlight::RenderingHelper#render_markdown`.

**Review by chrisrlc — APPROVED** [review]
Great tests!

**Pull request merged**